packages/clarity-js/src/data/metadata.ts

```diff
@@ -71,17 +81,27 @@ export function consent(): void {
 }
 
 export function clear(): void {
-  // Clear any stored information in the session so we can restart fresh the next time
-  if (supported(window, Constant.SessionStorage)) { sessionStorage.removeItem(Constant.StorageKey); }
+  // Clear any stored information in the session cookie so we can restart fresh the next time
```

Comment:
>session [](start = 41, length = 7)

session cookie is always a bit amgiuous between (1) a permanent cookie that hold the session id and (2) a session-scoped (lifetime) cookie. Reword to clarify which you mean.

Reply:
Sounds good. Will reword.

Reply:
Done